README.md

```diff
@@ -0,0 +1,221 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  <!-- - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖Hello Rails Backend App <a name="about-project"></a>
+
+A project to demonstrate setting up a Ruby on Rails project as a backend only app.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client (<a href="https://github.com/emmiiorji/react-rails-frontend-setup">Client Repository</a>)</summary>
+  <ul>
+    <li><a href="https://legacy.reactjs.org/docs/getting-started.html">React</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Demonstrate creating API-only rails app**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a> -->
+
+<!-- > Add a link to your deployed project. -->
+
+<!-- - [Live Demo Link](https://budget-app-8nzw.onrender.com) -->
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- Database Entity Relationship Diagram-->
+
+## 📍 Database Entity Relationship Diagram <a name="database-erd"></a>
+
+- [Link to the Database ERD](https://user-images.githubusercontent.com/20554583/221377937-7bd1e849-ad9b-442d-b7de-50b5cebd52db.png)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- A command line interface
+- Install [Ruby](https://www.ruby-lang.org/)
+- Install [PostgreSQL database](https://www.postgresql.org/)
+- Have Rails set up and running (`gem install rails`)
+
+### Setup
+
+- Clone this repository to your desired folder:
+    ```sh
+    cd my-folder
+    git clone git@github.com:emmiiorji/hello-react-rails.git
+    ```
+- Create a `.env` file in the root directory with your database configuration details following the `.env.example` file as guide
+
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my_blog
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails db:create
+  rails db:migrate
+  rails server
+  rails db:seed
+```
```

Comment:
- [ OPTIONAL ] I think all rails `db` commands should be run first to set up the database before you run the server